### PR TITLE
Implement player name

### PR DESCRIPTION
**Pull request by BertovDev**
Client -> https://github.com/lambdaclass/curse_of_mirra/pull/1578

### CODE REVIEW

**Review by agustinesco — COMMENTED** [review]
LGTM! will approve after the client pr got approved

**Review by agustinesco — APPROVED** [review]

**Review by manucamejo — APPROVED** [review]

**Pull request merged**
